root.tf

```diff
@@ -267,6 +267,18 @@ module "ecr_transfer_service_repository" {
   common_tags = local.common_tags
 }
 
+module "ecr_draft_metadata_validator_repository" {
+  source           = "./da-terraform-modules/ecr"
+  repository_name  = "draft-metadata-validator"
+  image_source_url = "https://github.com/nationalarchives/tdr-draft-metadata-validator/blob/main/Dockerfile"
+  allowed_principals = [
+    "arn:aws:iam::${data.aws_ssm_parameter.intg_account_number.value}:role/tdr-draft-metadata-validator-intg",
+    "arn:aws:iam::${data.aws_ssm_parameter.staging_account_number.value}:role/tdr-draft-metadata-validator-staging",
+    "arn:aws:iam::${data.aws_ssm_parameter.prod_account_number.value}:role/tdr-draft-metadata-validator-prod"
```

Comment:
Can you please use the Cammel case for the role name because we are using the camel case only for all the roles?
i.e. `TDRDraftMetadataValidatorIntg`

Reply:
Ignore this one, please.

Reply:
I just noticed that you need to append `-role` for the roles.
![image](https://github.com/nationalarchives/tdr-terraform-backend/assets/104834445/6d4b3c8c-a003-4900-8e08-8364dd89b5cd)

Reply:
thanks for spotting that